src/VisualStudio/Core/Def/Implementation/EditAndContinue/Interop/NativeMethods.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.EditAndContinue.Interop
+{
+    internal static class NativeMethods
+    {
+        private static Guid IID_IUnknown = Guid.Parse("00000000-0000-0000-C000-000000000046");
```

Comment:
Didn't know about that Guid ctor...seems approximately as readable, so I'll use it.  Thanks!